Command/ConsumeCommand.php

```diff
@@ -98,11 +98,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $message .= '.</info>';
         $output->writeln($message);
 
-        // AsyncQueueConsumer is immortal apart from Ctrl+C
-        if (!$this->endpoint->getConsumer() instanceof AsyncQueueConsumer) {
```

Comment:
Because you deleted this if, now shouldn't be needed the use reference to ```use Smartbox\Integration\FrameworkBundle\Components\Queues\AsyncQueueConsumer;```

Reply:
Ok. I removed the useless use.